src/components/Quote.vue

```diff
@@ -9,9 +9,9 @@ components.
 </docs>
 
 <template>
-	<router-link :to="{ hash, params: { skipLeaveWarning: true } }"
-		class="quote"
-		:class="{'quote-own-message': isOwnMessageQuoted}"
+	<component :is="component.tag"
+		:to="component.link"
+		:class="['quote', { 'quote-own-message': isOwnMessageQuoted }]"
```

Comment:
To separate static actual element classes and dynamically declared. (Saw it in many places recently)

```suggestion
		class="quote"
		:class="{ 'quote-own-message': isOwnMessageQuoted }"
```

Reply:
I found it more readable 🙈 
Reverted